src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -268,10 +267,9 @@ import { PageType } from 'shared/userSession/PageType';
 import { FeatureFlagEnum } from 'shared/featureFlags';
 import intersect from 'fast_array_intersect';
 import { PillStore } from 'shared/components/PillTag/PillTag';
+import { Simulate } from 'react-dom/test-utils';
```

Comment:
is this getting used?

Reply:
Fixed in [b64b0c0](https://github.com/cBioPortal/cbioportal-frontend/pull/4552/commits/b64b0c00389c30a4cc4f51e7ed12968ad17bcbe9)